hole/hole.go

```diff
@@ -116,15 +116,17 @@ func Play(ctx context.Context, holeID, langID, code string) (score Scorecard) {
 		}
 	}
 
+	const maxLength = 1024 * 1024
```

Comment:
Can we add a comment so I don't have to think about this value is every time I see it? Also I agree with https://github.com/code-golf/code-golf/issues/201#issuecomment-627123714 let's just copy TIO, 128 KiB ought to be enough for anybody ;-)

So with that I think something like this should be clear:
```go
const maxLength = 128 * 1024    // 128 KiB
```

Reply:
If it's not enough, we can bump it to 640 KB.